lib/fastlane/plugin/wpmreleasetoolkit.rb

```diff
@@ -4,7 +4,7 @@ module Fastlane
   module Wpmreleasetoolkit
     # Return all .rb files inside the "actions" and "helper" directory
     def self.all_classes
-      Dir[File.expand_path('**/{actions,helper,actions/configure,actions/android,helper/android,actions/ios,helper/ios}/*.rb', File.dirname(__FILE__))]
+      Dir[File.expand_path('**/{actions,helper,actions/common,actions/configure,actions/android,helper/android,actions/ios,helper/ios}/*.rb', File.dirname(__FILE__))]
```

Comment:
Can this just be?

```
Dir[File.expand_path('**/{actions,helper}/**/*.rb', File.dirname(__FILE__))]
```

I don't think we need to list every subfolder.

Reply:
👍